src/importexport/musicxml/internal/musicxml/export/exportmusicxml.cpp

```diff
@@ -897,23 +898,29 @@ void SlurHandler::doSlurs(const ChordRest* chordRest, Notations& notations, XmlW
     for (int i = 0; i < 2; ++i) {
         // search for slur(s) starting or stopping at this chord
         for (const auto& it : chordRest->score()->spanner()) {
+            String tagName = u"slur";
             auto sp = it.second;
-            if (sp->generated() || sp->type() != ElementType::SLUR || !ExportMusicXml::canWrite(sp)) {
+            if (sp->generated() || (sp->type() != ElementType::SLUR && sp->type() != ElementType::HAMMER_ON_PULL_OFF)
```

Comment:
Would `isSlur()` be appropriate here? (besides slurs and HOPO, that would also allow ElementType::TAPPING_HALF_SLUR, not sure if that's a problem)

Reply:
Let's try and see what happens.